addons/mail/static/src/components/chat_window_manager/chat_window_manager_tests.js

```diff
@@ -1464,6 +1464,245 @@ QUnit.test('chat window: switch on TAB', async function (assert) {
     );
 });
 
+QUnit.test('chat window: cycle between more than 2 chat windows (LTR)', async function (assert) {
```

Comment:
_"TAB cycle with 3 open chat windows (LTR)"_
Shorter and much more precise of what test does.

Same remark on test naming for the (RTL) version.